test/cmd/legacy-script.sh

```diff
@@ -850,7 +850,7 @@ runTests() {
     kube::test::get_object_assert 'clusterroles -l test-cmd=auth' "{{range.items}}{{$id_field}}:{{end}}" 'testing-CR:'
 
     failure_message=$(! kubectl auth reconcile "${kube_flags[@]}" -f test/fixtures/pkg/kubectl/cmd/auth/rbac-v1beta1.yaml 2>&1 )
-    kube::test::if_has_string "${failure_message}" 'only rbac.authorization.k8s.io/v1 is supported'
+    kube::test::if_has_string "${failure_message}" 'no matches for kind "ClusterRole" in version "rbac.authorization.k8s.io/v1beta1"'
```

Comment:
`reconcile` doesn't need restmapper info, it pins to v1 RBAC types and makes v1 API calls explicitly.

I think this might be a better change:

```diff
diff --git a/staging/src/k8s.io/kubectl/pkg/cmd/auth/reconcile.go b/staging/src/k8s.io/kubectl/pkg/cmd/auth/reconcile.go
index 3ead45bf644..5151310ace3 100644
--- a/staging/src/k8s.io/kubectl/pkg/cmd/auth/reconcile.go
+++ b/staging/src/k8s.io/kubectl/pkg/cmd/auth/reconcile.go
@@ -138,6 +138,7 @@ func (o *ReconcileOptions) Complete(cmd *cobra.Command, f cmdutil.Factory, args
 		NamespaceParam(namespace).DefaultNamespace().
 		FilenameParam(enforceNamespace, o.FilenameOptions).
 		Flatten().
+		Local().
 		Do()
 
 	if err := r.Err(); err != nil {
```